practices/structured-data.md

```diff
@@ -0,0 +1,70 @@
+# Structured data
+
```

Comment:
You _did_ ask for grammars and style, so you only have yourself to blame for what follows 😉 

You've used the _present continuous_ ("ing" verbs, e.g. "describing", "classifying", "providing") a lot. Switching to the _simple present_ will scan a little easier (e.g. "information that describes and classifies", "provides SEO benefit").